### PR TITLE
LPS-154916 Test fix for Content Dashboard Integration Tests after changes in files

### PR DESCRIPTION
**Pull request by YurenaC**
Motivation
=======
Fix failing test in content dashboard after changes made in the portal
[Link to story or ticket](https://issues.liferay.com/browse/LPS-154916)

Proposed Solution
========
As a new document (tree.png) is added by default when the portal starts, our Content Dashboard tests should be aware, in their checks, that the default amount of items is not zero anymore.

Steps to Verify:
----------------
1.Run the test in content-dashboard-test module and check that all passed.

Change summary:
---------------
- LPS-154916 Test fixing after changes in LPS-147440

### CODE REVIEW

**Comment by YurenaC** [comment]
ci:test:sf

**Comment by YurenaC** [comment]
ci:test:relevant

File: modules/apps/content-dashboard/content-dashboard-test/src/testIntegration/java/com/liferay/content/dashboard/web/internal/item/selector/test/ContentDashboardFileExtensionItemSelectorViewTest.java [hunk 1]
```diff
@@ -131,13 +131,20 @@ public void testGetData() throws Exception {
...
 
 	@Test
-	public void testGetDataWithNoFileEntries() throws Exception {
+	public void testGetDataWithoutNewFileEntries() throws Exception {
+		Map<String, Object> defaultData = _getData();
+
+		JSONArray defaultFileExtensionGroupsJSONArray =
+			(JSONArray)defaultData.get("fileExtensionGroups");
+
 		Map<String, Object> data = _getData();
```
Comment (cgoncas) [inline modules/apps/content-dashboard/content-dashboard-test/src/testIntegration/java/com/liferay/content/dashboard/web/internal/item/selector/test/ContentDashboardFileExtensionItemSelectorViewTest.java:140]: We are running getData twice right? This doesn't make a lot of sense :(, maybe we can delete this test.

File: modules/apps/content-dashboard/content-dashboard-test/src/testIntegration/java/com/liferay/content/dashboard/web/internal/portlet/test/ContentDashboardAdminPortletTest.java [hunk 2]
```diff
@@ -500,7 +503,8 @@ public void testGetSearchContainerWithDefaultOrder() throws Exception {
 		SearchContainer<Object> searchContainer = _getSearchContainer(
 			_getMockLiferayPortletRenderRequest());
 
-		Assert.assertEquals(2, searchContainer.getTotal());
+		Assert.assertEquals(
+			defaultSearchContainer.getTotal() + 2, searchContainer.getTotal());
```
Comment (cgoncas) [inline modules/apps/content-dashboard/content-dashboard-test/src/testIntegration/java/com/liferay/content/dashboard/web/internal/portlet/test/ContentDashboardAdminPortletTest.java:507]: Could we obtain defaultSearchContainer value, without calling the searchContainer, calling directly to a DLFileEntry method?

File: modules/apps/content-dashboard/content-dashboard-test/src/testIntegration/java/com/liferay/content/dashboard/web/internal/portlet/test/ContentDashboardAdminPortletTest.java [hunk 3]
```diff
@@ -849,10 +852,20 @@ public void testGetSearchContainerWithMultipleAuthors() throws Exception {
...
+				user.getUserId(), _group.getGroupId(), 0);
+			JournalArticle journalArticle2 = JournalTestUtil.addArticle(
+				_user.getUserId(), _group.getGroupId(), 0);
+
 			SearchContainer<Object> searchContainer = _getSearchContainer(
 				mockLiferayPortletRenderRequest);
 
-			Assert.assertEquals(2, searchContainer.getTotal());
+			Assert.assertEquals(
+				defaultSearchContainer.getTotal() + 2,
```
Comment (cgoncas) [inline modules/apps/content-dashboard/content-dashboard-test/src/testIntegration/java/com/liferay/content/dashboard/web/internal/portlet/test/ContentDashboardAdminPortletTest.java:867]: Could we obtain defaultSearchContainer value, without calling the searchContainer, calling directly to a Journal method?

**Review by cgoncas — CHANGES_REQUESTED** [review]
Thanks a lot for the PR.

Following Liferay StyleBook, we call the initialCount, initialCount, and the last result actualCount, do you mind updating the PR with that change?

**Comment by YurenaC** [comment]
Reviewing @cgoncas comments

File: modules/apps/content-dashboard/content-dashboard-test/src/testIntegration/java/com/liferay/content/dashboard/web/internal/portlet/test/ContentDashboardAdminPortletTest.java (hunk 2 above)
Comment (YurenaC) [reply modules/apps/content-dashboard/content-dashboard-test/src/testIntegration/java/com/liferay/content/dashboard/web/internal/portlet/test/ContentDashboardAdminPortletTest.java:507]: I've been looking at this and I am not sure that we could do it differently (easily). As to make it work in a general way, what we need to do is retrieve everything that is already there (currently is just a doc but in the future could be something else (or less) and I'd like this test to be flexible enough to be able to manage that)

What do you think about the `getCompanyEntriesCount(..)` method in the AssetEntryService? Not sure if this is too elaborated just for a test or maybe is not accurate enough
Let me know your thoughts :)

Comment (liferay-tango) [reply modules/apps/content-dashboard/content-dashboard-test/src/testIntegration/java/com/liferay/content/dashboard/web/internal/portlet/test/ContentDashboardAdminPortletTest.java:507]: The problem is that we not only support Asset, so we can keep it that way :)

**Comment by YurenaC** [comment]
ci:test:sf

**Comment by YurenaC** [comment]
ci:test:relevant

**Comment by YurenaC** [comment]
ci:forward

**Pull request closed without merging**